evm: Investigate Forge CI failures

e.g. on a Solana-only PR:
https://github.com/wormhole-foundation/example-native-token-transfers/actions/runs/8436531270/job/23104192465?pr=319


```
 Suite result: ok. 7 passed; 0 failed; 0 skipped; finished in 2.11ms (1.55ms CPU time)
2024-03-26T13:10:04.022444Z ERROR foundry_evm_core::fork::init: It looks like you're trying to fork from an older block with a non-archive node which is not supported. Please try to change your RPC url to an archive node if the issue persists.

Ran 1 test for test/IntegrationRelayer.t.sol:TestEndToEndRelayer
[FAIL. Reason: setup failed: Could not instantiate forked environment with fork url: https://bsc-testnet.public.blastapi.io] setUp() (gas: 0)
Suite result: FAILED. 0 passed; 1 failed; 0 skipped; finished in 784.07ms (0.00ns CPU time)


...


Suite result: FAILED. 33 passed; 1 failed; 0 skipped; finished in 30.85s (30.92s CPU time)

Ran 13 test suites in 36.73s (62.57s CPU time): 119 tests passed, 2 failed, 0 skipped (121 total tests)

Failing tests:
Encountered 1 failing test in test/IntegrationRelayer.t.sol:TestEndToEndRelayer
[FAIL. Reason: setup failed: Could not instantiate forked environment with fork url: https://bsc-testnet.public.blastapi.io] setUp() (gas: 0)

Encountered 1 failing test in test/NttManager.t.sol:TestNttManager
[FAIL. Reason: backend: failed while inspecting] test_maxOutTransceivers() (gas: 0)

Encountered a total of 2 failing tests, 119 tests succeeded
make: *** [Makefile:35: test-evm] Error 1
Error: Process completed with exit code 2.

```

--- a/evm/test/IntegrationManual.t.sol
+++ b/evm/test/IntegrationManual.t.sol
@@ -38,7 +38,7 @@ contract TestRelayerEndToEndManual is IntegrationHelpers, IRateLimiterEvents {
     IWormhole wormhole = IWormhole(0x68605AD7b15c732a30b1BbC62BE8F2A509D74b4D);
 
     function setUp() public {
-        string memory url = "https://bsc-testnet.public.blastapi.io";
+        string memory url = "https://bsc-testnet-rpc.publicnode.com";
         vm.createSelectFork(url);
         initialBlockTimestamp = vm.getBlockTimestamp();
 

